src/main/drivers/serial_softserial.c

```diff
@@ -227,6 +227,12 @@ serialPort_t *openSoftSerial(softSerialPortIndex_e portIndex, serialReceiveCallb
     IO_t rxIO = IOGetByTag(tagRx);
     IO_t txIO = IOGetByTag(tagTx);
 
+	if (tagRx == tagTx) {
+		if ((mode & MODE_RX) && (mode & MODE_TX)) {
```

Comment:
This line would be better written `if (mode & (MODE_RX | MODE_TX)) {`

Reply:
No this changes logic.

"If" clause should be executed if _both_ MODE_RX and MODE_TX flags are set.

if (mode & (MODE_RX | MODE_TX))
is executed if MODE_RX _or_ MODE_TX (or both) are set.

I can rewrite it to:
if ( (mode & ( MODE_RX | MODE_TX ) ) == (MODE_RX | MODE_TX) ) 

but I think original code is better.

Reply:
You're right, my bad :)